products/amazon-documentdb.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Amazon DocumentDB
+category: service
+tags: amazon database
+iconSlug: amazondocumentdb
+permalink: /amazon-documentdb
+releasePolicyLink: https://docs.aws.amazon.com/documentdb/latest/developerguide/compatibility.html
+releaseDateColumn: true
+releaseColumn: true
+
+releases:
+-   releaseCycle: "5.0"
+    releaseDate: 2023-03-01
+    eol: false
+    eoes: false
```

Comment:
```suggestion
```

`eoes` is for the "extended support column", which is not displayed here. So all `eoes` fields can be removed.